src/Eloquent/Relations/BelongsToManyOfDescendants.php

```diff
@@ -23,8 +23,8 @@ class BelongsToManyOfDescendants extends BelongsToMany
      * Create a new belongs to many of descendants relationship instance.
      *
      * @param \Illuminate\Database\Eloquent\Builder<TRelatedModel> $query
-     * @param \Illuminate\Database\Eloquent\Model $parent
-     * @param string $table
+     * @param TRelatedModel $parent
```

Comment:
Any doc like this that is defined on a parent should use `@inheritDoc` to avoid duplication

Reply:
Done where possible. The constructors cannot use `@inheritDoc` I think, since their signature is different